### ISSUE TITLE
Starting up MAGpy

### ISSUE THREAD

**Issue by naturepoker** [WatsonLab/MAGpy#47]
I'm sure this is an issue with me having never used snakemake before, but here goes:

I followed the steps on the github readme (test and building diamond database all worked perfectly), and am now trying to start the actual MAGpy process using the command:

`
snakemake -rp -s MAGpy --use-conda MAGpy --cores 8
`

My MAGs are in the mags directory in the cloned github repo.

The process fails with below message:

`
Building DAG of jobs...
MissingRuleException:
No rule to produce MAGpy (if you use input functions make sure that they don't raise unexpected exceptions).
`

Could you point me in the right direction? 
Thank you!

**Comment by gaworj** [comment]
Hi, 
I have encountered exactly the same problem and don't know how to solve it.

Bests,
Jan

**Comment by PeterCx** [comment]
I am having the exact same issue also. Did anyone find a workaround for this? Many thanks

Kind regards,

P

**Comment by mw55309** [comment]
Hi 

Sorry for the delay

I think the command should be

```sh
snakemake -rp -s MAGpy --use-conda --cores 8
```

Mick

**Comment by PeterCx** [comment]
Hi Mick,

By running this command does MAGpy do everything. i.e run CheckM for quality control, build the phylogenetic tree etc etc all in one command? Many thanks

Kind regards,

P

**Comment by mw55309** [comment]
Yes, that should run everything - if you have the databases installed first (see README)

**Comment by PeterCx** [comment]
This is working perfect. Thanks very much
